Source/SG/SG.cs

```diff
@@ -68,31 +73,27 @@ public struct PAENConfiguration
             public double CommandEnableTime_s;
             public double CommandDisableTime_s;
             public string PAEnableTriggerExportTerminal;
-            
-            public void SetDefaults()
+        }
+        public static PAENConfiguration GetDefaultPAENConfiguration()
```

Comment:
another opportunity for a static GetDefault method

Reply:
Resolving due to comment above.